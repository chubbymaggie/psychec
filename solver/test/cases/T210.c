void f() {
    a = 10;
    (void)a;
}

void g() {
    b = 10;
    (ANY)b;
}
