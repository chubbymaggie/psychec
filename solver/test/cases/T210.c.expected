/* Forward declarations */

/* Type definitions */
typedef  int /*<<< orphan*/  ANY ;

/* Variables and functions */
 int a ;
 int b ;

